docs/using-docker.rst

```diff
@@ -10,20 +10,18 @@ Install Docker onto the machine you plan to run Teleirc from.
 Which image do I choose?
 ************************
 
-Node Slim, Official Node Alpine Linux, and Fedora images are provided (ordered ascending by size).
+Official Node Alpine Linux, and Fedora images are provided (ordered ascending by size).
 
 +-----------------------------------------------------------------------------+-----------------------+---------+
 | Image                                                                       | File                  | Size    |
 +=============================================================================+=======================+=========+
 | `Node Alpine Linux <https://hub.docker.com/r/_/node/>`_ (``node:6-alpine``) | ``Dockerfile.alpine`` | 66.9 MB |
```

Comment:
I noticed we're using the `node:8-alpine` image now. Could you please update the reference here too?

Reply:
Sure.